CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Implement Visitor Design pattern in QueryBuilder to enable the capability to traverse through the complex QueryBuilder tree. ([#10110](https://github.com/opensearch-project/OpenSearch/pull/10110))
 - Provide service accounts tokens to extensions ([#9618](https://github.com/opensearch-project/OpenSearch/pull/9618))
 - Configurable merge policy for index with an option to choose from LogByteSize and Tiered merge policy ([#9992](https://github.com/opensearch-project/OpenSearch/pull/9992))
+- Implement Identity Plugin authentication support for Subjects ([#10614](https://github.com/opensearch-project/OpenSearch/pull/10614))
```

Comment:
Are the other entries preceded with a space before the '-' character, if so could we keep it consistent?

Reply:
Fixed